frontend/public/components/api-explorer.tsx

```diff
@@ -548,6 +534,34 @@ const APIResourceAccessReview: React.FC<APIResourceTabProps> = ({
       transforms: [sortable],
     },
   ];
+
+  const getSubjectType = (type: string) => {
```

Comment:
I'd call this `getSubjectTypeLabel` to make it clear it's the display value and not the actual value from the resource.

```suggestion
  const getSubjectTypeLabel = (type: string) => {
```